### DISCUSSION TITLE
Error with MOOSE installation on INL machine for BISON building.

### DISCUSSION THREAD

**Discussion by lyon69FR** [idaholab/moose#20413] [Q&A Getting Started] [answered]
Hi everyone,

I am currently trying to build BISON on INL machine SAWTOOTH3.
I followed the procedure (with PETSc module version 3.11.4-GCC) given in: https://mooseframework.inl.gov/bison/getting_started/hpc_new_clone.html.
All steps ran successfully until I built BISON. All of the errors displayed are reported on the file attached.

The error may stem from MOOSE installation. This is why I followed option 3 in  the procedure: 
https://mooseframework.inl.gov/docs/PRs/20068/site/getting_started/installation/inl_hpc_install_moose.html
This time too occurred an error, reported on the file attached.

Do you have any idea about this error coming from ?
Thank you in advance for your answer.
Vincent GAUTHIER.
[building_bison.docx](https://github.com/idaholab/moose/files/8145409/building_bison.docx)

**Comment by GiudGiud** [comment, marked as answer]
Hi Vincent

If you really do want to use the conda install, and suffer the performance penalty, you will have to purge that environment as you seem to have a broken mamba cache. This usually stems from a user interrupt in previous mamba commands. 

The preferred way would be for you to install your own petsc. The petsc module is really old. I updated this PR, this is now option 2 on that webpage you linked.

Guillaume

**Reply by lyon69FR** [reply]
Hi Guillaume,

I tried to install my own petsc with option 2, but I ended up with the same error message when I built BISON. (in the file attached).
[Building PETSc.docx](https://github.com/idaholab/moose/files/8147327/Building.PETSc.docx)

**Reply by GiudGiud** [reply]
let's just set up a call and check what's wrong. If that works for you, can you please email me some of your availabilities this week?
